spyder/config/main.py

```diff
@@ -22,7 +22,7 @@
 from spyder.plugins.editor.utils.findtasks import TASKS_PATTERN
 from spyder.utils.introspection.module_completion import PREFERRED_MODULES
 
-
+# 

```

Comment:
pep8speaks issue:

* In the file [`spyder/config/main.py`](https://github.com/spyder-ide/spyder/blob/109e236470b80ca588d5f7c0894ec455dad65014/spyder/config/main.py):

> [Line 25:2](https://github.com/spyder-ide/spyder/blob/109e236470b80ca588d5f7c0894ec455dad65014/spyder/config/main.py#L25): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace

Reply:
Fixed.